src/sakit/unused_level_select.c

```diff
@@ -62,7 +62,7 @@ void sub_80096DC(void)
     }
 }
 
-static void Task_8009780(void)
+static void UnusedLevelSelectPoll(void)
```

Comment:
Task functions should be named Task_ :)

Reply:
Also, as these are static you probably don’t need to include the “UnusedLevelSelect” bit as context in the module